cli/lib/api/search.ts

```diff
@@ -7,7 +7,7 @@ interface SearchResult {
   tags: string[]
 }
 
-export const search = async (searchTerm : string, searchVersion: string, baseUrl: string) : Promise<string|undefined> => {
+export const search = async (searchTerm : string, searchVersion: string, baseUrl: string) : Promise<string|undefined|any> => {
```

Comment:
I'm generally hesistant on any types for stuff, did a type get all dumb on you?

Reply:
I see the issue here, If we update the `SearchResult` interface to include `owner: string` you can remove both `any` and clear your errors.

Reply:
I updated the SearchResult interface to now include owner (and the `about` field was missing too), that removed the typing issue.